docs/pages/connect-your-client/gui-clients.mdx

```diff
@@ -116,7 +100,10 @@ your GUI client below. When entering the hostname, use `localhost` rather than
 `127.0.0.1`.
 </TabItem>
 <TabItem label="Separate ports">
-If you're not using TLS routing, run the following command to see the database
+
+(!docs/pages/includes/determinetls.mdx!)
+
+If you're not using [TLS routing](../management/operations/tls-routing.mdx), run the following command to see the database
```

Comment:
```suggestion
If you're not using [TLS Routing](../management/operations/tls-routing.mdx), run the following command to see the database
```

Reply:
Also, as mentioned in another comment, I think it would be  smoother to introduce TLS Routing at the top of these instructions in one sentence and remove this link. This way, the user wouldn't need to leave the page.